src/CairoMakie.jl

```diff
@@ -494,6 +494,35 @@ function draw_plot(scene::Scene, screen::CairoScreen, primitive::Combined)
     end
 end
 
+function rgbatuple(c::Colorant)
+    rgba = RGBA(c)
+    red(rgba), green(rgba), blue(rgba), alpha(rgba)
+end
+
+"""
+Special method for polys so we don't fall back to atomic meshes, which are much more
+complex and slower to draw than standard paths with single color.
+"""
+function draw_plot(scene::Scene, screen::CairoScreen, poly::Poly)
+    if poly.visible[] == false
```

Comment:
Is this needed as a special case, or would it be better to check for `visible` even in non-atomic plot types?